src/ad-services/connatix/index.ts

```diff
@@ -32,13 +50,57 @@ export class Connatix extends BaseServiceSetup {
 			return Promise.resolve();
 		}
 
-		communicationService.on(eventsRepository.AD_ENGINE_UAP_LOAD_STATUS, (action: UapLoadStatus) => {
-			if (!action.isLoaded) {
-				utils.logger(logGroup, 'No Fan Takeover loaded - injecting Connatix player');
-				initConnatixHeadScript(this.cid);
-				utils.logger(logGroup, 'Connatix head script is ready');
-				this.playerInjector.insertPlayerContainer(this.cid);
+		utils.logger(logGroup, 'initialized', this.cid);
+
+		communicationService.on(
+			eventsRepository.AD_ENGINE_UAP_LOAD_STATUS,
+			this.loadOnUapStatus.bind(this),
+		);
+	}
+
+	private loadOnUapStatus({ isLoaded, adProduct }: UapLoadStatus) {
+		if (!isLoaded && adProduct !== 'ruap') {
```

Comment:
Instead, I'd suggest reverse this condition and return early to enhance readability

Reply:
Yeah, let's leave it as-is so it matches the logic in Anyclip. However, it's fishy - I've reached out to Seba to find out the business use-case.

Reply:
It was not about changing the logic. This is what I meant:

```suggestion
        if (isLoaded || adProduct === 'ruap') {
            utils.logger(logGroup, 'Connatix blocked because of Fan Takeover');
            return;
        }
        // rest of the code
```

Reply:
Upon reversing the condition, it becomes apparent that the current log message lacks clarity.